library/Makefile

```diff
@@ -88,8 +88,22 @@ OBJS_CRYPTO=	aes.o		aesni.o		arc4.o		\
 		psa_its_file.o					\
 		ripemd160.o	rsa_internal.o	rsa.o  		\
 		sha1.o		sha256.o	sha512.o	\
-		threading.o	timing.o	version.o	\
-		version_features.o		xtea.o
+		threading.o	timing.o			\
+		xtea.o
+
+# For files generated by the parent project (Mbed TLS) when building Mbed
+# Crypto as a submodule, ensure that the parent project instance is used.
+ifdef USE_CRYPTO_SUBMODULE
```

Comment:
I'd recommend changing that to an `ifneq ($(USE_CRYPTO_SUBMODULE), 0)`, same as here:
https://github.com/ARMmbed/mbedtls/pull/2403/commits/fc5a7f53154057f25273eafaf58b7b3b2cd112ee
since USE_CRYPTO_SUBMODULE=0 will also have it defined.